README.md

```diff
@@ -210,6 +210,22 @@ Note:
 - spike only support rv64* bare-metal/elf toolchain.
 - gdb simulator only support bare-metal/elf toolchain.
 
+#### Testing GCC, Binutils, and glibc of a Linux toolchain
+
+The default Makefile target to run toolchain tests is `report`.
+This will run all tests of the GCC regression test suite.
+Alternatively the following command can be used to do the same:
+
+    make check-gcc-linux
```

Comment:
`check-gcc`, it will point to check-gcc-linux if `--enable-linux`

Reply:
ok